tools/gencode.ts

```diff
@@ -117,13 +117,15 @@ const extractHandlersMetadata: () => Promise<
   GencodeIpcHandler[]
 > = async () => {
   // Load list of file paths in handlers directory, recursively visiting subfolders
-  const fileNames = (await recursivelyReadDirectory(HANDLERS_DIRECTORY)).map(
-    (filePath: string) => filePath.split(`${HANDLERS_DIRECTORY}/`)[1]
+  const filePaths = (await recursivelyReadDirectory(HANDLERS_DIRECTORY)).map(
```

Comment:
Looks like this depends on https://github.com/chloebrett/mlvet/pull/295 so wait for that to be merged first

Reply:
https://github.com/chloebrett/mlvet/pull/295 has been merged into develop. Please merge develop into your branch. There will probably be a merge conflict with gencode as I cleaned it before merging. Please just choose develop.